src/Raven.Server/ServerWide/Commands/ClusterTransactionCommand.cs

```diff
@@ -610,43 +613,65 @@ private static unsafe SingleClusterDatabaseCommand ReadCommand<TTransaction>(Tra
 
         public override object FromRemote(object remoteResult)
         {
-            var errors = new List<ClusterTransactionErrorInfo>();
-            if (remoteResult is BlittableJsonReaderArray array)
+            if (remoteResult is BlittableJsonReaderObject bjro)
             {
-                foreach (var o in array)
-                {
-                    if (o is not BlittableJsonReaderObject blittable)
-                        continue;
-
-                    errors.Add(ToClusterTransactionErrorInfo(blittable));
-                }
+                return ToClusterTransactionResult(bjro);
+            }
 
-                return errors;
+            if (remoteResult is BlittableJsonReaderArray bjra)
+            {
+                return GetErrors(bjra);
             }
+
             return base.FromRemote(remoteResult);
         }
 
+        private static ClusterTransactionResult ToClusterTransactionResult(BlittableJsonReaderObject bjro)
+        {
+            var result = new ClusterTransactionResult();
```

Comment:
Maybe `JsonDeserializationCluster` can be used here

Reply:
Done